pkg/udf/map_udf.go

```diff
@@ -130,9 +132,16 @@ func (u *MapUDFProcessor) Start(ctx context.Context) error {
 	}
 
 	maxMessageSize := sharedutil.LookupEnvIntOr(dfv1.EnvGRPCMaxMessageSize, sdkclient.DefaultGRPCMaxMessageSize)
-
 	enableMapUdfStream := sharedutil.LookupEnvBoolOr(dfv1.EnvMapStreaming, false)
+	enableBatchMapUdf := sharedutil.LookupEnvBoolOr(dfv1.EnvBatchMap, false)
+
+	// We can have the vertex running only of the map modes
+	if enableMapUdfStream && enableBatchMapUdf {
```

Comment:
Improvement: Shall we use one `server-info` for all the mappers, and start the mode based what is written in the file?

Reply:
We can do that, Can take up in a followup PR?

Reply:
https://github.com/numaproj/numaflow/issues/1810